packages/core/src/index.ts

```diff
@@ -13,6 +13,7 @@ export * from "./middleware"
 export * from "./types"
 export * from "./supertokens"
 export * from "./passport-adapter"
+export * from "./db-utils"
```

Comment:
Change this to explicitly re-export the items that need exported (this index file is refactored on master to follow this pattern)